src/v/redpanda/application.h

```diff
@@ -173,7 +189,7 @@ class application {
     ss::sharded<features::feature_table> _feature_table;
     ss::sharded<kafka::group_manager> _group_manager;
     ss::sharded<kafka::group_manager> _co_group_manager;
-    ss::sharded<net::server> _rpc;
+    ss::sharded<rpc::rpc_server> _rpc;
```

Comment:
switching to rpc_server could be a separate commit

Reply:
Done